OnlineTeamScan_V2_BE/BL/Services/IndividualScoreServices/IndividualScoreService.cs

```diff
@@ -28,7 +28,12 @@ public IndividualScoreService(IUnitOfWork unitOfWork, IMapper mapper)
 
         public IndividualScoreReadDto GetIndividualScoreById(int id)
         {
-            return _mapper.Map<IndividualScoreReadDto>(_unitOfWork.IndividualScoreRepository.GetById(id));
+            var individualScore = _unitOfWork.IndividualScoreRepository.GetById(id);
+
+            if (individualScore == null)
```

Comment:
Eventueel if (name is null) gebruiken?

Ik ben geen .NET expert, maar dat lijkt me safer.